README.md

```diff
@@ -1,44 +1,81 @@
 # ovh-api-services
 
-![OVH component](https://user-images.githubusercontent.com/3379410/27423240-3f944bc4-5731-11e7-87bb-3ff603aff8a7.png)
-
-[![NPM](https://nodei.co/npm/ovh-api-services.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/ovh-api-services/)
-
-[![Maintenance](https://img.shields.io/maintenance/yes/2018.svg)]() [![Chat on gitter](https://img.shields.io/gitter/room/ovh/ux.svg)](https://gitter.im/ovh/ux)
-
 > Contains all $resource for API.
 
-## Table of contents
+[![Downloads](https://badgen.net/npm/dt/ovh-api-services)](https://npmjs.com/package/ovh-api-services) [![Dependencies](https://badgen.net/david/dep/ovh-ux/ovh-api-services)](https://www.npmjs.com/package/ovh-api-services?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/ovh-api-services)](https://www.npmjs.com/package/ovh-api-services?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
 
-- [Installation](#installation)
-- [Usage](#usage)
 
-
-## Installation
+## Install
 
 ```sh
 $ yarn add ovh-api-services
 ```
 
-- Add Angular dependency "ovh-api-services"
-- Load files via wiredep, or manually
+## Usage
 
-This will also download the dependencies.
+```js
+import angular from 'angular';
+import 'ovh-api-services';
+
+angular
+  .module('myApp', [
+    'ovh-api-services',
+  ]);
+```
+
+### Services
+
+All services must return an [AngularJS $resource](https://docs.angularjs.org/api/ngResource/service/$resource) and can be called by using:
+
+
+| Service   | Engine  | Library                                            |
+|-----------|---------|----------------------------------------------------|
+| `v6`      | APIv6   | n/a                                                |
```

Comment:
Are you sure though ? I've seem some `V6` previously, I don't know if this is still the case !

Reply:
Seen together. It was a misunderstood between the injection of `v6` key and the `V6` suffix used across all services registration.